services/device-registry-base/src/main/java/org/eclipse/hono/service/management/device/DeviceDto.java

```diff
@@ -53,12 +61,13 @@ public DeviceDto() {
      *
      * @return A DTO instance for creating a new entry.
      */
-    public static <P extends Device, T extends DeviceDto> T forCreation(final Supplier<T> supplier, final String tenantId, final String deviceId, final Device device, final String version) {
+    public static <P extends Device, T extends DeviceDto> T forCreation(final Supplier<T> supplier, final String tenantId, final String deviceId, final Boolean autoProvisioned, final Device device, final String version) {
```

Comment:
there doesn't seem to be any *P* involved in this method, or am I mistaken?

Reply:
No, you're right. Thanks for spotting 👍